dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/ItemRestRepository.java

```diff
@@ -88,4 +90,28 @@ public ItemResource wrapResource(ItemRest item, String... rels) {
         return new ItemResource(item, utils, rels);
     }
 
+    @Override
+    protected void delete(Context context, UUID id) throws AuthorizeException {
+        Item item = null;
+        try {
+            item = is.find(context, id);
+            if (is.isInProgressSubmission(context, item)) {
+                throw new UnprocessableEntityException("The item cannot be deleted. "
+                        + "It's part of a in-progress submission.");
+            }
+            if (item.getTemplateItemOf() != null) {
+                throw new UnprocessableEntityException("The item cannot be deleted. "
+                        + "It's a template for a collection");
+            }
+        } catch (SQLException e) {
+            throw new RuntimeException(e.getMessage(), e);
+        }
+        try {
+            is.delete(context, item);
+            is.removeAllBundles(context, item);
```

Comment:
this is not needed as done internally in the ItemService.delete
https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/java/org/dspace/content/ItemServiceImpl.java#L649

Reply:
Done